### PR TITLE
jqueryui/draggable wrapper

### PR DESCRIPTION
**Pull request by jrefano**
as this is about to be identical in poet and pro2-ui, i decided to move it here

### CODE REVIEW

**Comment by jrefano** [comment]
@bejudged retest

**Comment by mikesherov** [comment]
Missing tests.

**Comment by mikesherov** [comment]
failing tests

File: trait/draggable.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+define([
```
Comment (mikesherov) [inline trait/draggable.js]: belongs in `ui/trait/draggable`

File: ui/trait/draggable.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+define([
+  'jqueryui/draggable'
+], function(draggable) {
+  'use strict';
+  return {
+    makeDraggable: function($view, handle, containment, cancelSelector) {
+      var cancelDefault;
+
+      $view = $view || this.$view;
```
Comment (mikesherov) [inline ui/trait/draggable.js]: this line isn't being tested though.

File: test/specs/ui/trait/draggable.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      this.$view.makeDraggable(this.$view);
+
+      expect(this.$view.draggable('option', 'handle')).toEqual('.js-drag-handle');
+      expect(this.$view.draggable('option', 'containment')).toEqual('window');
+      expect(this.$view.draggable('option', 'cancel')).toEqual(this.$view.cancelDefault + ', .js-drag-cancel');
+    });
+
+    it('sets the specified options', function() {
+      this.$view.makeDraggable(this.$view, '.test-handle', '.test-containment', '.test-cancel');
+      expect(this.$view.draggable('option', 'handle')).toEqual('.test-handle');
```
Comment (mrjoelkemp) [inline test/specs/ui/trait/draggable.js]: Newline for consistency

File: ui/trait/draggable.js (hunk 2 above)
Comment (mikesherov) [inline ui/trait/draggable.js]: now it is

File: ui/trait/draggable.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+define([
+  'jqueryui/draggable'
+], function(draggable) {
+  'use strict';
+  return {
```
Comment (mrjoelkemp) [inline ui/trait/draggable.js]: Non-blocker: newline before

**Comment by mrjoelkemp** [comment]
:+1:

**Comment by mikesherov** [comment]
:+1:

**Pull request merged**